src/core/sock/sock-redirect.cpp

```diff
@@ -118,15 +119,6 @@ template <typename T> void assign_dlsym(T &ptr, const char *name)
         }                                                                                          \
     }
 
-#define VERIFY_PASSTROUGH_CHANGED(__ret, __func_and_params__)                                      \
-    do {                                                                                           \
-        bool passthrough = p_socket_object->isPassthrough();                                       \
-        __ret = __func_and_params__;                                                               \
-        if (!passthrough && p_socket_object->isPassthrough()) {                                    \
-            handle_close(__fd, false, true);                                                       \
-        }                                                                                          \
-    } while (0);
-
 void get_orig_funcs()
```

Comment:
Now that this is behind the XLIO_STATIC_BUILD flag, then xlio_init (which calls this) no longer compiles.

Reply:
Fixed